scripts/user_control.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
```

Comment:
Why is this script sitting in the root directory? Also, best practice is to keep scripts as short as possible, and to have large source files separate, with scripts only importing/invoking them.

Reply:
Maybe start a "controls" folder in common, with this as one of the controllers.